examples/MultipleRedeemers.purs

```diff
@@ -0,0 +1,254 @@
+module Examples.MultipleRedeemers (threeRedeemerContract) where
+
+import Prelude
+
+import Cardano.Types.Value
+  ( CurrencySymbol
+  , NonAdaAsset(NonAdaAsset)
+  , Value(Value)
+  , mkCurrencySymbol
+  )
+import Contract.Aeson (decodeAeson, fromString)
+import Contract.Monad
+  ( ContractConfig(ContractConfig)
+  , Contract
+  , liftContractM
+  , liftedE
+  , liftedM
+  , logInfo'
+  , runContract
+  , traceContractConfig
+  )
+import Contract.PlutusData (PlutusData, unitDatum, unitRedeemer)
+import Contract.Prelude (liftM, log, mconcat, unwrap, wrap)
+import Contract.Prim.ByteArray (byteArrayFromAscii)
+import Contract.ScriptLookups as Lookups
+import Contract.Scripts (Validator, validatorHash, MintingPolicy)
+import Contract.Transaction
+  ( BalancedSignedTransaction(BalancedSignedTransaction)
+  , balanceAndSignTx
+  , submit
+  )
+import Contract.TxConstraints as Constraints
+import Contract.Utxos (utxosAt)
+import Contract.Value (mkTokenName, scriptCurrencySymbol)
+import Contract.Value as Value
+import Contract.Wallet (mkNamiWalletAff)
+import Data.Array (replicate)
+import Data.BigInt (fromInt)
+import Data.Foldable (length, sum)
+import Data.Bitraversable (bitraverse)
+import Data.Either (hush)
+import Data.Int (toNumber)
+import Data.Map as Map
+import Data.Set as Set
+import Data.Maybe (Maybe)
+import Data.Traversable (traverse)
+import Data.Tuple (Tuple(Tuple), fst, snd)
+import Effect.Aff (Aff, delay, Milliseconds(Milliseconds), error)
+import Effect.Aff.Class (liftAff)
+import Plutus.ToPlutusType (toPlutusType)
+import Plutus.Types.Address (scriptHashAddress)
+import Plutus.Types.Transaction (UtxoM, Utxo)
+import Plutus.Types.CurrencySymbol (getCurrencySymbol)
+import Types.Scripts (ValidatorHash)
+import Types.TokenName (TokenName)
+import Types.Transaction (TransactionHash, TransactionInput)
+
+-- FIXME: this doesn't work without a browser
+threeRedeemerContract :: Aff Unit
+threeRedeemerContract = do
+  log "creating utxos"
+  ContractConfig defaults <- traceContractConfig
+  wallet <- mkNamiWalletAff
+  let
+    cfg :: ContractConfig ()
+    cfg = ContractConfig $ defaults { wallet = pure wallet }
+
```

Comment:
Why not have a single `Contract` action and run it here with `runContract_`? As it is you're calling `runContract` twice -- once to get the tuple and then to build and submit the transaction

Reply:
I wanted to differentiate between two contracts. One to create tokens and one to spend those tokens. 
I think that makes things clearer and easier to understand.

Reply:
In that case, my recommendation would be to split the top-level function up and then just have:

```purescript
createTokens :: Contract () (TransactionHash /\ Int)

spendTokens ::  Contract () Unit
```

I think that would help clarify what's going on a bit, then both of them can be called within a single `runContract_`

Reply:
I'll do it like that then :)